packages/admin/src/pages/build-application/[applicationId]/components/Sections.tsx

```diff
@@ -12,15 +12,24 @@ interface SectionsProps {
   sections: ApplicationFormSection[];
   applicationId: string;
   applicationStatus: ApplicationFormSummary['applicationStatus'];
+  resolvedUrl: string;
+  csrfToken: string;
 }
 
 const Sections = ({
   sections,
   applicationId,
   applicationStatus,
+  resolvedUrl,
+  csrfToken,
 }: SectionsProps) => {
   return (
-    <>
+    <FlexibleQuestionPageLayout
```

Comment:
Worried this adds some vertical padding to the page, just make sure there's a normal amount of space above this component plz 🙏

Reply:
Looks to be fine, the find-a-grant paragraph before the table is normally space